data/hai/hai reveal 0 prologue.txt

```diff
@@ -0,0 +1,734 @@
+			label sayari
+			`	"My point is that if these Hai were really interesting in invading us, do you think we'd all be here right now? The Hai are not hostile to us, so why would we give them a reason to be by sitting here and bugging their warship captains?" Remington says to the crowd.`
+			`	You look over to the group of Hai to see Sayari speaking to them in their own language, likely trying to prevent the Hai from doing anything brash.`
+			`	Remington speaks with the crowd of humans for a little while longer, eventually convincing them to leave the Hai alone. "Let's head back to <planet>," Remington says. "I've seen enough here. And we should probably think about having the spaceport authority here restrict who can land, at least until all this blows over."`
+			`	"Perhaps," Sayari responds.`
+	on visit
+		dialog phrase "generic missing stopover or passengers"
+	on complete
+		conversation
+			`You return to the briefing room at the secure port with Remington and Sayari. "This Hai holographic tech is pretty nifty isn't it?" Remington says while playing around with the device he used on <planet stopovers>. "Way more advanced than anything we've got back home. It's also pretty useful that there's a port for one of our cables, otherwise I never would have figured out how to get that Pug image on there."`
+			`	"Ah, yes," Sayari says. "Many decades of interaction with humans has led us to accommodate much of our technology to your needs. Humans are our guests here, so we saw it only right to make you feel more comfortable. A shame to see a few bad apples spoil the bunch."`
+			`	"You have apples up here?" Remington asks.`
+			`	Sayari chuckles. "Just as we have adapted out technology to human needs, we have also imported many human foods, and learned human sayings. It has all been very exciting."`
+			`	Sayari pauses for a moment. "Our civilization is tens of thousands of years old. There is little in that time that we have not done on our own. But seeing such a young species has led to much excitement. I wish for that excitement to not turn into strife. This is what I explained to the Hai on <planet stopovers>. 'These humans are young. Be patient with them.'"`
+			choice
+				`	"Glad to see that you still have a positive view of us."`
+				`	"Well we're here to do our best to make that wish come true."`
+			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn the human language, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`
```

Comment:
No, I'd think there would certainly be more than one language. HR did have a lot of mentions of "Galactic Standard" once but I had removed all those mentions since it was strange that nobody else in the rest of the game ever mentions it.

It could be changed to this:
```suggestion
			`	Sayari goes on about the history between Hai and humanity, how a colony exploration team sanctioned by the Republic found the wormhole in Ultima Thule and made first contact between the Hai and Republic. Early on, they used the Quarg as translators between the two parties, but eventually the Hai began to learn human languages, and even some humans began to learn Hai, although it seems much more difficult for a human to speak Hai than the other way around.`
```

Reply:
Is there a standard language, or just human translation devices everywhere?

Reply:
I think there would be auxiliary languages, but this is also 3000AD humanity. Second is probably very likely.

Reply:
I always got the impression there was one distinct "primary" human language, with others hanging on for sub/cultural relevance. From the Hai perspective, especially in a diplomatic context, referring to this primary language as "the human language" would indeed be appropriate.

The Hai probably still have at least a few of their own old cultural languages hanging around, after all, even if just in academic settings.